pageserver/src/tenant.rs

```diff
@@ -983,6 +994,8 @@ impl Tenant {
             }
         }
 
+        let mut shared_state_builder = timeline::detach_ancestor::SharedStateBuilder::default();
```

Comment:
would prefer if this variable name contained `detached_ancestor`

Reply:
Could you elaborate what is this?

Reply:
This no longer exists, I just put a hashmap there.